app/uk/gov/hmrc/residencenilratebandcalculator/ServiceBindings.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.residencenilratebandcalculator
+
+import play.api.inject.{Binding, Module}
+import play.api.mvc.{DefaultPlayBodyParsers, PlayBodyParsers}
+import play.api.{Configuration, Environment}
+
+class ServiceBindings extends Module {
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
+    bindUtil
+  }
+
+  private def bindUtil = Seq(
+//    bind(classOf[PlayBodyParsers]).to(classOf[DefaultPlayBodyParsers]).eagerly()
```

Comment:
I don't understand what's happening in this file. "bindings" calls bindUtil which just returns an empty sequence.

Reply:
Will remove